### DISCUSSION TITLE
Steven Lynch here. How do I run the latest versions of fheroes2?

### DISCUSSION THREAD

**Discussion by EveS-INFINTTeY-ALnL-22-000-369** [ihhub/fheroes2#4152] [Q&A]
Toned down a bit. ;) :P :$ How do I set up mingw or whatever to run fheroes2? I have GOG.com Heroes 2 Gold.

**Comment by oleg-derevenetz** [comment]
Hi @EveS-INFINTTeY-ALnL-22-000-369 please take a look at the [installation guide](https://ihhub.github.io/fheroes2/INSTALL.html).
